backend/migrations/20241216163427_first-kemomimi.down.sql

```diff
@@ -0,0 +1,11 @@
+-- 製品-カテゴリ関係テーブル（product_category）削除
+DROP TABLE product_category;
+
+-- 分類テーブル（category）削除
+DROP TABLE category;
+
+-- 備品テーブル（public_item）削除
+DROP TABLE public_item;
+
+-- 製品テーブル（product）削除
+DROP TABLE product;　
```

Comment:
```suggestion
DROP TABLE product;
```
downのほうは今試したのですが、全角スペースでエラーです。

Reply:
すみません、除去しました